pkg/reconciler/reconciler.go

```diff
@@ -18,32 +19,39 @@ import (
 
 const defaultDebounceTime = 20 * time.Second
 
+type Adapter interface {
```

Comment:
If keep Adapter as the name here, implementations of it should also be called Adapter, not Reconciler.

Reply:
Sorry, fixed that.